### ISSUE TITLE
The --shell-escape requirement makes it difficult to use with an IDE.

### ISSUE THREAD

**Issue by MakisH** [waltsims/Thesis_Template_CSE#5]
Usually, when using a TeX IDE (e.g. Texmaker, Atom/LaTeX_Tools), the user expects the template to run just by invoking the build command. The template shouldn't ask from the user to change their workflow (e.g. use the terminal) but we also want to provide the Minted package, that requires the special `--shell-escape` flag. Unfortunately, not all the editors are flexible enough for this.

However, I think I have the solution!

**Comment by waltsims** [comment]
What ide do you use without that option? I know Atom's LaTeX plugin has it

**Comment by MakisH** [comment]
I am using Atom with latex-tools, but it either doesn't work or it is not obvious how to set it up. See also https://github.com/msiniscalchi/atom-latextools/issues/35.

**Comment by waltsims** [comment]
There is a checkbox in the atom package preferences for shell escape. I have tried it myself. See also here: [atom-latex/issues/3](https://github.com/thomasjo/atom-latex/issues/3)

**Comment by MakisH** [comment]
I was refering to the [latex-tools](https://atom.io/packages/latextools) package and not to the [latex](https://atom.io/packages/latex). The latex-tools doesn't have a specific option for that and setting the flag manually didn't work, as refered to the above-mentioned issue.

It looks like that option was specific to the latex-tools. I replaced it with information on how to make it work, both in the `main.tex` and in the `README.md`.

Note: Although the `% !TEX option` is specific to latex-tools, the `% !TEX root` that I used in another Pull Request is recognised by [more editors](http://tex.stackexchange.com/a/84687/27863).
